app/assets/stylesheets/shared/form.scss

```diff
@@ -235,7 +235,17 @@
 }
 
 .ss-main.form-select.is-invalid:not([multiple]):not([size]) {
+  padding-right: $form-select-feedback-icon-padding-end;
+  background-image: url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 16 16'%3e%3cpath fill='none' stroke='%23323233' stroke-linecap='round' stroke-linejoin='round' stroke-width='2' d='m2 5 6 6 6-6'/%3e%3c/svg%3e"),
+    url("data:image/svg+xml,%3csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 12 12' width='12' height='12' fill='none' stroke='%23dc3545'%3e%3ccircle cx='6' cy='6' r='4.5'/%3e%3cpath stroke-linejoin='round' d='M5.8 3.6h.4L6 6.5z'/%3e%3ccircle cx='6' cy='8.2' r='.6' fill='%23dc3545' stroke='none'/%3e%3c/svg%3e");
```

Comment:
I tried leveraging the Sass-variable found here: https://getbootstrap.com/docs/5.3/forms/select/#sass-variables `$form-select-indicator: ` but when I used it, the images all went blank

But the image never appeared.